src/Database/Database.php

```diff
@@ -622,7 +639,21 @@ public function renameAttribute(string $collection, string $id, string $name): b
             }
         }
 
+        foreach ($indexes as $index) {
+            $indexAttributes = $index->getAttribute('attributes', []);
+            $indexAttributes = \array_map(function($attribute) use($id, $name) {
+                if($attribute === $id) {
+                    return $name;
+                }
+
+                return $attribute;
+            }, $indexAttributes);
```

Comment:
```suggestion
            $indexAttributes = \array_map(fn($attribute) => (attribute === $id) ? $name : $attribute , $indexAttributes);
```

Arrow functions allow you to avoid the `use` keyword.

Reply:
Updated